azure_arc_k8s_jumpstart/aro/dockerfile

```diff
@@ -1,9 +1,8 @@
-FROM mcr.microsoft.com/azure-cli:latest
+FROM mcr.microsoft.com/azure-cli:2.18.0
```

Comment:
any specific reason for the pindown version?

Reply:
Yep, less maintenance